tasks/upgrade.yml

```diff
@@ -61,6 +61,17 @@
     loop_var: volume_item
   when:  cluster_info.ovirt_clusters[0].gluster_service | bool
 
+- name: Refresh gluster heal info entries to database
+  uri:
+    url: "{{ ovirt_auth.url }}/clusters/{{ cluster_info.ovirt_clusters[0].id }}/refreshglusterhealstatus"
+    method: POST
+    body_format: json
+    validate_certs: false
+    headers:
+      Authorization: "Bearer {{ ovirt_auth.token }}"
+    body: "{}"
+  when: cluster_info.ovirt_clusters[0].gluster_service | bool
+
```

Comment:
That's super ugly. Could you please post a patch to ovirt_cluster module to expose this action properly?

Reply:
This feature was added just last month [ClusterService.java](https://github.com/oVirt/ovirt-engine-api-model/blob/master/src/main/java/services/ClusterService.java#L426).
I'm not sure what to do with it. Probably revert changes of the previous PR and wait till 4.4 will become a minimal requirement. What do you think @mwperina?

Reply:
@mwperina My engine code is going merge in next few day's @Ori Liel Working on it he facing some issue for next version release. I will do the changes in ovirt_cluster module and raise PR. Whenever sdk4.4 will release will update this particular task.. In Cluster Upgrade Role.

Reply:
OK, let's have it as right now and let's fix it once within ovirt-ansible-collection